<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xiugaidoc_wid</class>
 <widget class="QWidget" name="xiugaidoc_wid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_text">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>30</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改医生数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>邮箱：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_tel">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电话：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_idno">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_work_no">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*工号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_keshi">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>科室：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_keshi">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>门诊部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>内科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>外科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>儿科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>妇科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>神经科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>眼科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>耳鼻喉科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>口腔科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>泌尿科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>骨科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>创伤科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>内分泌科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>心脏外科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>麻醉科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>皮肤科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>传染病科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>脑外科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>胸外科</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_work_no">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_idno">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>身份证：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
